imagestreams/mysql-rhel.json

```diff
@@ -63,36 +46,36 @@
         }
       },
       {
-        "name": "8.0-el7",
+        "name": "8.0-el9",
         "annotations": {
-          "openshift.io/display-name": "MySQL 8.0 (RHEL 7)",
+          "openshift.io/display-name": "MySQL 8.0 (RHEL 9)",
           "openshift.io/provider-display-name": "Red Hat, Inc.",
-          "description": "Provides a MySQL 8.0 database on RHEL 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mysql-container/blob/master/README.md.",
-          "iconClass": "icon-mysql-database",
-          "tags": "mysql",
+          "description": "Provides a MySQL 8.0 database on RHEL 9. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mysql-container/blob/master/8.0/README.md.",
+          "iconClass": "icon-mysql",
+          "tags": "database,mysql",
           "version": "8.0"
         },
         "from": {
           "kind": "DockerImage",
-          "name": "registry.redhat.io/rhscl/mysql-80-rhel7:latest"
+          "name": "registry.redhat.io/rhel9/mysql-80:latest"
         },
         "referencePolicy": {
           "type": "Local"
         }
       },
       {
-        "name": "8.0",
```

Comment:
This is a nonstandard tag, it should be added to the config like this: https://github.com/sclorg/httpd-container/blob/19edd655824b9d03d4a6e0d509fce02fdaf0c457/imagestreams/imagestreams.yaml#L38-L41

Reply:
Fixed by https://github.com/sclorg/mysql-container/pull/371/commits/17717fbe5b2ff1838a81adf88859e1542dc47b78